package ua.good.network

/**
 * Неизвестная ошибка
 */
class ErrorResponseException(message: String) : Throwable(message)
